docs/Contributing/Testing-and-local-development.md

```diff
@@ -340,7 +340,7 @@ Fleet's `docker-compose` file includes a SAML identity provider (IdP) for testin
 
 ### Configuration
 
-Configure SSO on the Organization Settings page with the following:
+Configure SSO on the **Integration settings** page with the following:
```

Comment:
This change already exists on the `docs-v4.67.0` branch, I have no idea why it's showing up in this diff. Please ignore.

(This change will also request a codeowner review since it's in `/docs/Contributing`, but I will merge this PR after approving anyway since the change already exists and `docs-v4.67.0` is not a protected branch.)